test/functional/page_objects/common_page.js

```diff
@@ -270,6 +270,19 @@ export function CommonPageProvider({ getService, getPageObjects, getPageObject }
     async isChromeVisible() {
       return await testSubjects.exists('kibanaChrome');
     }
+
+    async isTopNavVisible() {
+      return await testSubjects.exists('top-nav');
+    }
+
+    async waitForTopNavToBeVisible() {
```

Comment:
I think the timeout in the default exists function is much shorter because we don't want to hang for 10 seconds if the thing doesn't exist but that's what we expect.  So the differences between the two implementations (current vs the one you suggest) is one relies on the default timeout for retry.try and the other is passing in an arbitrary timeout.  I suppose both are really arbitrary timeouts, but I think it's better using retry.try instead of passing in a new number for someNewTimeout.  Then any changes to the retry.try timeout affect all tests using it.  I can inline isTimeNavVisible since no other function is using it though.